File: Data Structures/Java/Lists/DoubleNode.java
package Java.Lists;

public class DoubleNode<E> {
    
    public DoubleNode<E> previous;
    public DoubleNode<E> next;
    private E data;

    public DoubleNode(E data){
        this.data = data;
        previous = null;
        next = null;
    }

    public E getData(){
        return data;
    }
}

File: Data Structures/C#/Lists/DoubleNode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lists
{
    public class DoubleNode<E>
    {
        public DoubleNode<E> previous { get; set; }
        public DoubleNode<E> next { get; set; }
        public E data { get; }
        public DoubleNode(E data)
        {
            this.data = data;
            previous = null;
            next = null;
        }
    }
}
